packages/devtools_app/test/test_data/cpu_profile_test_data.dart

```diff
@@ -729,6 +792,8 @@ final CpuStackFrame stackFrameC2 = CpuStackFrame(
   category: 'Dart',
   rawUrl:
       'file:///path/to/flutter/packages/flutter/lib/src/widgets/binding.dart',
+  processedUrl:
+      'file:///path/to/flutter/packages/flutter/lib/src/widgets/binding.dart',
```

Comment:
should this be using `package:flutter/...` format? Same comment applies to other processedUrl fields that begin with `file:///` in this fake test data

Reply:
yes probably. this was me being lazy learning about the specs initially 😓